extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -47,160 +74,366 @@ Because of standardization of markup, support for multiple providers, and improv
+    <td>&lt;object&gt;</td>
+    <td>Free-form data which can be used for template rendering, e.g. messaging related to metering or article count. See <a href="#customising-content">Customising Content</a> for more details.</td>
+  </tr>
+</table>
+
+Example response for a Reader who is a subscriber and is logged into their account:
 
-## Service configuration
+```js
+{
+  "granted": true,
+  "grantReason": "SUBSCRIBER",
+  "data" : {
+    "isLoggedIn": true
+  }
+}
+```
 
-The `amp-subscriptions` extension must be configured using JSON configuration:
+Example response for an anonymous Reader who has read 4 out of 5 free articles:
 
-```html
-<script type="application/json" id="amp-subscriptions">
-  {
-    "services": [
-      {
-        // Service 1 (local service)
-      },
-      {
-        // Service 2 (a vendor service)
-      }
-    ],
-    "score": {
-      "supportsViewer": 10,
-      "isReadyToPay": 9
-    },
-    "fallbackEntitlement": {
-      "source": "fallback",
-      "granted": true,
-      "grantReason": "SUBSCRIBER/METERING",
-      "data": {...}
-    }
+```js
+{
+  "granted": true,
+  "grantReason": "METERING",
+  "data" : {
+    "isLoggedIn": false,
+    "articlesRead": 4,
+    "articlesLeft": 1,
+    "articleLimit": 5
   }
-</script>
+}
 ```
 
-The `services` property contains an array of service configurations. There must be one "local" service and zero or more vendor services.
+Example response for an anonymous Reader who does not have access because they have read 5 out of 5 free articles:
 
-If you'd like to test the document's behavior in the context of a particular viewer, you can add `#viewerUrl=` fragment parameter. For instance, `#viewerUrl=https://www.google.com` would emulate a document's behavior inside a Google viewer.
+```js
+{
+  "granted": false,
+  "data" : {
+    "isLoggedIn": false,
+    "articlesRead": 5,
+    "articlesLeft": 0,
+    "articleLimit": 5
+  }
+}
+```
 
-## Selecting a service
+Notice, while it's not explicitly visible, all vendor services also implement authorization endpoints of their own and conform to the same response format.
 
-If no service returns an entitlement that grants access, all services are compared by calculating a score for each and the highest scoring service is selected. Each service has a `"baseScore"` (default 0). A value < 100 in the `baseScore` key in any service configuration represents the initial score for that service. If no `baseScore` is specified it defaults to `0`.
+#### Login page
 
-The score is calculated by taking the `baseScore` for the service and adding dynamically calculated weights from `score[factorName]` configuration multiplied by the value returned by each service for that `factorName`. Services may return a value between [-1..1] for factors they support. If a service is not aware of a factor or does not support it `0` will be returned.
+This flow will be triggered as a result of a `"login"` action as described in the [Actions][12] section.
```

Comment:
I don't understand what we mean by flow?

Reply:
Added some more detail here.